notebooks/official/model_monitoring/batch_prediction_model_monitoring.ipynb

```diff
@@ -0,0 +1,800 @@
+{
```

Comment:
<p> you'd like to be alerted automatically and responsively-&gt; you can be alerted automatically</p><p> </p><p>so that you can anticipate problems before they affect your customer experiences or your revenue streams. -&gt; ; whereby, you can detect model decay which may negatively impact your operations -- such as your customer experience and/or revenue.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/723/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>fixed</p>